### PR TITLE
Css lint

### PR DESCRIPTION
**Pull request by SKOLZ**
# Summary

added the css linter, its working properly.

### CODE REVIEW

**Comment by SKOLZ** [comment]
@sbalay when you check this i want to ask you something

File: scss-lint.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    style: uppercase
+
+  LeadingZero:
+    enabled: true
+    style: include_zero
+
+  NameFormat:
+    enabled: true
+    allow_leading_underscore: true
+    convention: camel_case
```
Comment (sbalay) [inline scss-lint.yml]: This is for css classes names?

Comment (SKOLZ) [inline scss-lint.yml]: no, this is used for variables or even mixins. Like $LightBlue for example.

**Comment by sbalay** [comment]
:+1:

**Pull request merged**